content/en/containers/kubernetes/log.md

```diff
@@ -184,8 +206,35 @@ spec:
 # (...)
 ```
 
+#### Example log Autodiscovery Annotations
+
+The following Pod annotation defines the integration template for an example container. It was defined within the Pod template's annotations, rather than on the Deployment itself. This log configuration sets all the logs from the `app` container with the tags `source:java`, `service:example-app`, and the extra tag `foo:bar`.
```

Comment:
```suggestion
The following Pod annotation defines the integration template for an example container. It is defined within the Pod template's annotations, rather than on the Deployment itself. This log configuration sets all the logs from the `app` container with the tags `source:java`, `service:example-app`, and the extra tag `foo:bar`.
```